### ISSUE TITLE
problem using `template` in `tabAutocompleteOptions`

### ISSUE THREAD

**Issue by rjsr213** [continuedev/continue#852]
### Before submitting your bug report

- [ ] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [ ] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [ ] I've seen the [troubleshooting guide](https://continue.dev/docs/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- Continue: 0.9.62
```

### Description

Basically the problem is that when using `template` it sends the prompt to the model with a bad format
Example:
instead of sending `=` send this `&#x3D;`
This does not happen when not declaring `template`
